src/Admin/Views/Providers/Create.cshtml

```diff
@@ -3,14 +3,62 @@
     ViewData["Title"] = "Create Provider";
 }
 
+@section Scripts {
+    <script>
+        function toggleProviderTypeInfo(value) {
+            document.querySelectorAll('[id^="info-"]').forEach(el => { el.classList.add('d-none'); });
+            document.getElementById('info-' + value).classList.remove('d-none');
+        }
+    </script>
+}
+
 <h1>Create Provider</h1>
 
 <form method="post">
     <div asp-validation-summary="All" class="alert alert-danger"></div>
     
     <div class="form-group">
-        <label asp-for="OwnerEmail"></label>
-        <input type="text" class="form-control" asp-for="OwnerEmail">
+        <h2>Provider Type</h2>
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Type" name="Type" value="@((int)ProviderType.Msp)" id="radioMsp" onclick="toggleProviderTypeInfo(@((int)ProviderType.Msp))" checked>
+            <label class="form-check-label align-middle" for="radioMsp">
+                Managed Service Provider
+            </label>
+            <br/>
+            <label class="form-check-label small text-muted ml-3 align-top" for="radioMsp">
+                Access to clients organization
+            </label>
+        </div>
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Type" name="Type" value="@((int)ProviderType.Reseller)" id="radioReseller" onclick="toggleProviderTypeInfo(@((int)ProviderType.Reseller))">
+            <label class="form-check-label align-middle" for="radioReseller">
+                Reseller
+            </label>
+            <br/>
+            <label class="form-check-label small text-muted ml-3 align-top" for="radioReseller">
+                Access to clients billing
+            </label>
+        </div>
+    </div>
+    
+    <div id="info-0" class="form-group @(Model.Type != ProviderType.Msp ? "d-none" : string.Empty)">
```

Comment:
You can get rid of these conditional classes once you use `if` statements

Reply:
Both div elements need to be constructed independently of the `Model.Type` value, the only thing that changes is if it's meant to appear or not. If I switched to `if` statements I'd have to duplicate the code to have the same outcome.

Reply:
Nope fair enough, after reading your last comment I'm fine with how you've handled this